.gitignore

```diff
@@ -65,3 +65,7 @@ coverage.xml
 
 # Kedro
 *.log
+
+# Python
+.venv/
+mise.toml
```

Comment:
I've added this because I'm using [mise](https://mise.jdx.dev/lang/python.html) to manage my Python environment, not conda.

Reply:
I was a mise user as well for a while! Until I discovered https://docs.astral.sh/uv/ 😄